message_send_types_test.go

```diff
@@ -380,3 +380,63 @@ func TestSendDraftReq_AddPackage(t *testing.T) {
 		})
 	}
 }
+
+func TestSendDraftReq_AddClearPackage(t *testing.T) {
+	key, err := crypto.GenerateKey("name", "email", "rsa", 2048)
+	require.NoError(t, err)
+
+	kr, err := crypto.NewKeyRing(key)
+	require.NoError(t, err)
+
+	tests := []struct {
+		name    string
+		body    string
+		prefs   map[string]proton.SendPreferences
+		attKeys map[string]*crypto.SessionKey
+		wantErr bool
+	}{
+		{
+			name: "clear plain text with signature",
+			body: "this is a text/plain body",
+			prefs: map[string]proton.SendPreferences{"clear-plain-with-sig@email.com": {
+				Encrypt:          false,
+				SignatureType:    proton.DetachedSignature,
+				EncryptionScheme: proton.ClearScheme,
+				MIMEType:         rfc822.TextPlain,
+			}},
+			wantErr: false,
+		},
+		{
+			name: "clear plain text with bad scheme (error)",
+			body: "this is a text/plain body",
+			prefs: map[string]proton.SendPreferences{"clear-plain-with-sig@email.com": {
+				Encrypt:          false,
+				SignatureType:    proton.DetachedSignature,
+				EncryptionScheme: proton.PGPInlineScheme,
+				MIMEType:         rfc822.TextPlain,
+			}},
+			wantErr: true,
+		},
+		{
+			name: "clear rich text with signature (error)",
+			body: "this is a text/html body",
+			prefs: map[string]proton.SendPreferences{"clear-plain-with-sig@email.com": {
+				Encrypt:          false,
```

Comment:
Can clear text packages be encrypted?

Reply:
Here is the restriction for ClearText:
- EncryptionScheme: ClearScheme (means no encryption)
- SignatureType: DetachedSignature 
- MIMEType: text/plain

Reply:
So we could add a test where someone set `Encrypt` to true and it should produce an error?